src/util/cmdline.cpp

```diff
@@ -145,6 +148,42 @@ const char *cmdlinet::getval(const char *option) const
   return value->second.front().c_str();
 }
 
+std::string cmdlinet::expand_path_string(std::string path) const
+{
+  return path;
+}
+
+// Will return empty string if invalid.
+std::string cmdlinet::get_config_file_location() const
+{
+  const auto envloc = std::getenv("ESBMC_CONFIG_FILE");
```

Comment:
We don't load anything from env, all options are passed as flags to the binary. I don't mind keeping this but please add an option too.

Reply:
Well, the `ESBMC_OPTS` envvar does exist for while. When also adding an option the precedence should be documented:
1. ESBMC_OPTS
2. ESBMC_CONFIG_FILE
3. command line options

possibly with the additional note that duplicate/contradicting options in either will lead to ESBMC refusing to work, see also discussion #1916.

Reply:
The reason I included this one is that there might be users that would like to set a global config file from their ENV. Granted, with the inclusion of a config file, we don't have a reason to add any more env vars :)

> We don't load anything from env, all options are passed as flags to the binary. I don't mind keeping this but please add an option too.

So should the command line parameter be called `--config` with a short `-c` if it doesn't conflict with anything?